README.md

```diff
@@ -258,6 +258,15 @@ Types are exported from `./lib/types/[node/web]/index.d.ts` and should be automa
   }
   ```
 
+- `submitFundTransaction({ txId })` - Submits a fund transaction to the Turbo Payment Service for processing. The `txId` is the transaction ID of the transaction to be submitted.
```

Comment:
Nit: wording sounds like you're actually making a transaction. Suggestion:
`Submits the transaction ID of a funding transaction to Turbo Payment Service for top up processing."

Reply:
May also want to have this function's description as well as that of `topUpWithTokens` reference one another. e.g. for this one: "Use this API if you've already executed your token transfer to the Turbo wallet. Otherwise, consider using `topUpWithTokens` to execute a new token transfer to the Turbo wallet and submit its resulting transaction ID for top up processing all in one go."

Reply:
added readme updates for these comments here: c31d1d6